src/arc.ts

```diff
@@ -215,12 +215,14 @@ export async function initializeArc(provider?: any): Promise<boolean> {
   if (success) {
     provider = arc.web3.currentProvider; // won't be a string, but the actual provider
     // save for future reference
-    provider.__networkId = await getNetworkId(provider);
+    const networkId = await getNetworkId(provider);
```

Comment:
why do this and also still disable the require-atomic-updates the line below?

Reply:
yeah, that was a side-effect of two separate tries to eliminate the warning.  I'll consolidate.

Reply:
done